template/_package.json

```diff
@@ -42,15 +42,17 @@
     "@adonisjs/shield": "^1.0.4"<% } %><% if (ui === 'bootstrap') { %>,
     "bootstrap-vue": "^2.0.0-rc.11",
     "bootstrap": "^4.1.3"<% } else if (ui === 'vuetify') { %>,
-    "vuetify": "^1.5.5",
+    "vuetify": "^1.5.9",
     "vuetify-loader": "^1.2.1"<% } else if (ui === 'bulma') { %>,
     "@nuxtjs/bulma": "^1.2.1"<% } else if (ui === 'element-ui') { %>,
     "element-ui": "^2.4.11"<% } else if (ui === 'ant-design-vue') { %>,
     "ant-design-vue": "^1.1.10"<% } else if (ui === 'buefy') { %>,
     "nuxt-buefy": "^0.3.2"<% } else if (ui === 'iview') { %>,
     "iview": "3.1.5"<% } %><% if (axios === 'yes') { %>,
     "@nuxtjs/axios": "^5.3.6"<% } %><% if (pwa === 'yes') { %>,
-    "@nuxtjs/pwa": "^2.6.0"<% } %><% if (ui === 'tachyons') { %>,
+    "@nuxtjs/pwa": "^2.6.0"<% } %><% if (typescript === 'yes') { %>,
+    "ts-node": "^8.0.3",
```

Comment:
I think `typescript` is a peer dependency of it (for the compiler). /cc @kevinmarrec

Reply:
That's true, and as `typescript` is shipped in `@nuxt/typescript` which is installed as devDep, it will bring the issue that `typescript` won't be installed in production.